src/spatial.rs

```diff
@@ -58,30 +82,22 @@ impl SpatialAudio {
     }
 }
 
-pub(crate) fn run_spatial_audio(
-    spatial_audio: Res<SpatialAudio>,
-    receiver: Query<&GlobalTransform, With<AudioReceiver>>,
-    emitters: Query<(&GlobalTransform, &AudioEmitter)>,
-    mut audio_instances: ResMut<Assets<AudioInstance>>,
-) {
-    if let Ok(receiver_transform) = receiver.get_single() {
-        spatial_audio.update(receiver_transform, &emitters, &mut audio_instances);
-    }
-}
-
-pub(crate) fn cleanup_stopped_spatial_instances(
+fn cleanup_stopped_spatial_instances(
     mut emitters: Query<&mut AudioEmitter>,
     instances: ResMut<Assets<AudioInstance>>,
 ) {
-    for mut emitter in emitters.iter_mut() {
-        let handles = &mut emitter.instances;
-
-        handles.retain(|handle| {
-            if let Some(instance) = instances.get(handle) {
-                instance.handle.state() != kira::sound::PlaybackState::Stopped
-            } else {
-                true
-            }
+    emitters.iter_mut().for_each(|mut emitter| {
+        emitter.instances.retain(|handle| {
+            instances.get(handle).map_or(true, |instance| {
+                !matches!(instance.handle.state(), kira::sound::PlaybackState::Stopped)
+            })
         });
-    }
+    });
+}
+
+fn spatial_audio_enabled(
```

Comment:
This means the system won't run without the resource, even if all emitters have their own `SpacialRadius` components.
I think it would be more idiomatic to just let users add the `SpatialAudioPlugin`.